tests/console/java.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Description: Basic Java test
+# Summary: It installs every Java version which is available into
+#	   the repositories and then it performs a series of basic
+#          tests, such as verifying the version, compile and run
+#          the Hello World program
+# Maintainer: Panos Georgiadis <pgeorgiadis@suse.com>
+# Maintainer: Andrej Semen <asemen@suse.com>
+
+use base "opensusebasetest";
+use base "x11test";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use base "consoletest";
+
+
+
+sub run() {
+
+    my $self = shift;
```

Comment:
This will pass the first argument passed to run();

Is the same as taking the first argument from argv in other languages... 

http://perldoc.perl.org/functions/shift.html